src/Components/ListArticles.jsx

```diff
@@ -1,89 +1,106 @@
-import React, { Component } from 'react'
-import { getData } from '../Modules/RequestArticles'
-import { Container, Header, Item } from 'semantic-ui-react'
+import React, { Component } from 'react'
+import { getData } from '../Modules/RequestArticles'
+import { Container, Header, Item } from 'semantic-ui-react'
+import ViewArticle from './ViewArticle';
 
-class ListArticles extends Component {
-  state = {
-    articles: [],
-    error_message: null
-  }
+class ListArticles extends Component {
+  state = {
+    articles: [],
+    error_message: null,
+    renderArticle: false,
+    renderArticleId: null
+  }
 
-  componentDidMount() {
-    this.getArticles()
-  }
+  componentDidMount() {
+    this.getArticles()
+  }
 
-  makeIngress = (content, wordcount) => {
-    let ingress = content.split(' ').slice(0, wordcount).join(' ')
-    return ingress + ' ...'
+  setErrorMessage = (error) => {
+    this.setState({
+      error_message: error
+    })
   }
 
-  async getArticles() {
-    let result = await getData()
+  makeIngress = (content, wordcount) => {
+    let ingress = content.split(' ').slice(0, wordcount).join(' ')
+    return ingress + ' ...'
+  }
 
-    if (result.status === 400) {
-      this.setState({
-        error_message: result.error_message
-      })
-    } else {
-      this.setState({
-        articles: result
-      })
-    }
-  }
+  async getArticles() {
+    let result = await getData()
```

Comment:
Why are you extracting out the `getData()` function? You double the error handling because you do that in the module and you do that in here as well.
I see that you do that multiple times in the code, so we can go with this for now, but I would rethink this approach because you are adding additional bloat to the codebase.